idrstream/idr_stream.py

```diff
@@ -218,12 +218,12 @@ def prepare_batch(self, batch_metadata: pd.DataFrame):
             well_movie_path = self.downloader.download_image(
                 plate, well_num, download_save_path
             )
-            self.logger.info(f"well_movie_path: {well_movie_path}")
-            
+            self.logger.info(f"Movie downloaded to {well_movie_path}")
```

Comment:
this is fine for now, but this won't always be a movie, correct?

Reply:
Yep, this is correct. I think its good to keep the `well_movie_path` name for the variable so it is clear what needs to be changed in future versions of `IDR_stream`.